tasks/install_binary.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Get fabio checksums file
```

Comment:
I think you can replace this and the shell below with the uri module with return_content set to yes, you will need to ensure that httplib2 is installed first though.